index.js

```diff
@@ -25,12 +25,13 @@ function translate(text, opts) {
 
     opts.from = opts.from || 'auto';
     opts.to = opts.to || 'en';
+    opts.region = opts.region || "com";
```

Comment:
keep project pattern, you forgot here, use ' instead of " here ^^

Reply:
also, nice feat!

Reply:
ok, I know it